facebook/__init__.py

```diff
@@ -255,6 +256,12 @@ def request(
                 args["access_token"] = self.access_token
 
         try:
+
+            logging.info('GRAPH REQUEST URL:%s' % (FACEBOOK_GRAPH_URL + path))
+            logging.info('GRAPH REQUEST params:')
+            logging.info(args)
+            logging.info('GRAPH REQUEST data:')
+            logging.info(post_args)
```

Comment:
This should be removed from here and added in another pull request. Adding logging has nothing to do with error handling.